### PR TITLE
Move to assertj

### PR DESCRIPTION
**Pull request by eddumelendez**

### CODE REVIEW

File: core/src/jarFileTest/java/org/testcontainers/PublicBinaryAPITest.java [hunk 1]
```diff
@@ -41,9 +41,9 @@ public class PublicBinaryAPITest extends AbstractJarFileTest {
     private static String SHADED_PACKAGE_PATH = SHADED_PACKAGE.replaceAll("\\.", "/");
 
     static {
-        Assertions.registerFormatterForType(ClassNode.class, it -> it.name);
-        Assertions.registerFormatterForType(FieldNode.class, it -> it.name);
-        Assertions.registerFormatterForType(MethodNode.class, it -> it.name + it.desc);
+        registerFormatterForType(ClassNode.class, it -> it.name);
+        registerFormatterForType(FieldNode.class, it -> it.name);
+        registerFormatterForType(MethodNode.class, it -> it.name + it.desc);
```
Comment (bsideup) [inline core/src/jarFileTest/java/org/testcontainers/PublicBinaryAPITest.java:46]: not sure about this change, let's maybe revert? Makes it hard to follow who defines `registerFormatterForType`

File: examples/selenium-container/src/test/java/SeleniumContainerTest.java [hunk 2]
```diff
@@ -45,7 +45,7 @@ public void simplePlainSeleniumTest() {
         driver.get("http://host.testcontainers.internal:" + port + "/foo.html");
         List<WebElement> hElement = driver.findElementsByTagName("h");
 
-        assertTrue("The h element is found", hElement != null && hElement.size() > 0);
+        assertThat(hElement != null && hElement.size() > 0).as("The h element is found").isTrue();
```
Comment (bsideup) [inline examples/selenium-container/src/test/java/SeleniumContainerTest.java:48]: nit: I know that it was automated, but could probably use AssertJ's support for lists

**Review by bsideup — APPROVED** [review]
Fantastic job! 👏  Also... OpenRewrite FTW 💯

**Comment by kiview** [comment]
The leverage of automation, amazing! 🥳 🤯
